platform/ext/target/nordic_nrf/common/core/hw_init.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef HW_INIT_H__
+#define HW_INIT_H__
+
+#define _SCS_BASE_ADDR 0xE000E000
+#define _SCS_MPU_CTRL (_SCS_BASE_ADDR + 0xd94)
```

Comment:
These are prefixed with `_` and look internal. Do we need them in the headerfile?

Also, why isn't hw_init_reset_on_boot declared here?

Reply:
Yeah, so this header is only used by assembly. Which is why it only defines some constants from CMSIS. And does not declare the function.